staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/conversion.go

```diff
@@ -41,15 +41,22 @@ func Convert_apiextensions_JSON_To_v1_JSON(in *apiextensions.JSON, out *JSON, s
 	if err != nil {
 		return err
 	}
-	out.Raw = raw
+	if len(raw) == 0 || string(raw) == "null" {
+		// match JSON#UnmarshalJSON treatment of literal nulls
+		out.Raw = nil
+	} else {
+		out.Raw = raw
+	}
 	return nil
 }
 
 func Convert_v1_JSON_To_apiextensions_JSON(in *JSON, out *apiextensions.JSON, s conversion.Scope) error {
 	if in != nil {
 		var i interface{}
-		if err := json.Unmarshal(in.Raw, &i); err != nil {
-			return err
+		if len(in.Raw) > 0 && string(in.Raw) != "null" {
```

Comment:
Don't do the `string()` cast to avoid making a potentially huge temporary copy, checking the length & 4 characters manually is probably best but there are other options too.